src/ngraph_builder.cc

```diff
@@ -1916,6 +1916,97 @@ static Status TranslateFusedBatchNormGradOp(
   return Status::OK();
 }
 
+static Status TranslateFusedMatMulOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  int num_args;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "num_args", &num_args));
+
+  std::vector<string> fused_ops;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "fused_ops", &fused_ops));
+
+  auto CreateNgDot = [&](shared_ptr<ng::Node>& ng_lhs,
+                         shared_ptr<ng::Node>& ng_rhs,
+                         shared_ptr<ng::Node>& ng_dot) {
+
+    // Transpose arguments if requested.
+    bool transpose_a = false;
+    bool transpose_b = false;
+
+    if (GetNodeAttr(op->attrs(), "transpose_a", &transpose_a) == Status::OK() &&
+        transpose_a) {
+      ng_lhs = ng::builder::numpy_transpose(ng_lhs, ng::AxisVector{1, 0});
+    }
+    if (GetNodeAttr(op->attrs(), "transpose_b", &transpose_b) == Status::OK() &&
+        transpose_b) {
+      ng_rhs = ng::builder::numpy_transpose(ng_rhs, ng::AxisVector{1, 0});
+    }
+
+    // The default axis count for nGraph's Dot op is 1, which is just what
+    // we need here.
+    ng_dot = ConstructNgNode<ngraph::op::Dot>(op->name(), ng_lhs, ng_rhs);
+
+    return Status::OK();
+  };
+
+  // TODO: Need to add BiasAdd + Activation fusedMatMul op
+  /*if (!VecStrCmp(fused_ops, {"BiasAdd"})) {
+    return errors::Unimplemented("Unsupported _FusedMatMul " +
+                                 str_util::Join(fused_ops, ","));
+  }*/
```

Comment:
Remove this since you now have Relu support?

Reply:
Done 👍